CMakeLists.txt

```
@@ -22,9 +22,17 @@ include(PandoraCMakeSettings)
 # Prefer local include directory to any paths to installed header files
 include_directories(include)
 
+# Set up C++ Standard
+# ``-DCMAKE_CXX_STANDARD=<standard>`` when invoking CMake
+set(CMAKE_CXX_STANDARD 17 CACHE STRING "")
+
+if(NOT CMAKE_CXX_STANDARD MATCHES "17|20")
```

```suggestion
if(NOT CMAKE_CXX_STANDARD MATCHES "17")
```

It's c++20 that breaks what I attempted to fix in #17